src/validations/rules/ssp.sch

```diff
@@ -1537,6 +1537,33 @@
                 role="error"
                 test="current() = $security-sensitivity-levels">[Section B Check 3.10] A FedRAMP SSP must specify an allowed security sensitivity
                 level.</sch:assert>
+            <sch:let
+                name="securityLevelStr"
+                value="string-join(../o:security-impact-level//text())"/>
+            <sch:let
+                name="securityImpactLevel"
+                value="
+                if (matches($securityLevelStr, 'high'))
+                then
+                ('fips-199-high')
+                else
+                (if (matches($securityLevelStr, 'moderate'))
+                then
+                ('fips-199-moderate')
+                else
+                ('fips-199-low'))" />
```

Comment:
I see a different code formatting result when "Format and Indent" is applied to this section of code.

Are you using the settings described [here](https://github.com/18F/fedramp-automation/blob/develop/src/validations/CONTRIBUTING.md) in "Formatting XML"?

Reply:
This was missed during manual formatting.  Fixed now.